New package: VectorUtils v1.0.0

JuliaRegistrator opened the pull request:
- Registering package: VectorUtils
- Repository: https://github.com/mirajcs/VectorUtils.jl
- Created by: @mirajcs
- Version: v1.0.0
- Commit: 7418e2fbfdb583022b04e5048b01e7c9ff629d86
- Reviewed by: @mirajcs
- Reference: https://github.com/mirajcs/VectorUtils.jl/commit/7418e2fbfdb583022b04e5048b01e7c9ff629d86#commitcomment-170613235
- Description: A Julia package for symbolic and numeric vector operations.
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6e784f83e8ef28ab05ee3b05c87873198829a699a9a3142eda96cc34680512c12ba2c0ee7c168ae521fb6df592d44e8ae8ec152119921291ad8818dbeb41fcf959e88f51b88f7155af0ccd5ebfa58bd1c6b5a292b51b8b25d8701acd5eac72abd5d0c5d109ae15b82171f4415bced658a6271f5a8280d795bc0d6aa16e05b9ec5f9cd79c0b3d0e2aeb4df47c30cdbc30f60bb838b6fa4c5d2148f24f73d5064c0e6b4c0d654ac3efb45b3cac6d2006495 -->

gdalle commented:
Hi, congrats on the new package!
Can you think of a more specific name? VectorUtils might be a bit too broad and confusing

JuliaRegistrator committed:
New package: VectorUtils v1.0.0

UUID: ce5ad29b-8b1a-4d19-a814-cc15ce7eccd1
Repo: https://github.com/mirajcs/VectorUtils.jl.git
Tree: 61f2bc0e48bdaa5da492c59beaa0f858c6b44eb4

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3

gdalle commented:
The issue here is that every field of math and computer science uses vectors. From what I understand, your package is specifically targeting parametric curves, which is why it deserves a more specific name.
I saw that there are also generic vector-related functions in there, but most of those can be found in the [`LinearAlgebra` standard library](https://docs.julialang.org/en/v1/stdlib/LinearAlgebra/): see for example `norm`, `dot` and `cross`

goerz commented:
Closing in favor of https://github.com/JuliaRegistries/General/pull/142560.

Although I'm not completely sure that `GeometryToolkit` is that much more descriptive than `VectorUtils`. Note that both "Toolkit" and "Utils" are indicator-word for the lack of a well-defined scope for the package and usually are not an ideal suffix for a Julia package name. I'm not so concerned about this that I would block the registration, personally. Still: something to consider
